.github/workflows/protobuf.yml

```diff
@@ -0,0 +1,83 @@
+name: protobuf
+# CI tests to run against the protobuf schema on change:
+
+on:
+  # run for any PRs raising changes to the protobuf files or setup
+  pull_request:
+    paths:
+      - 'cylc/flow/network/protobuf/**'
+
+jobs:
+  protobuf:
+    runs-on: ubuntu-latest
+    timeout-minutes: 10
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+        with:
+          # we need all of the commits on the PR branch in order to be able to add new ones
+          fetch-depth: 100
+
+      - name: Configure git
+        uses: cylc/release-actions/configure-git@v1
+
+      - name: Install Protobuf
+        uses: mamba-org/setup-micromamba@v1
+        with:
+          # install protobuf into a mamba env (note use shell = "bash -el {0}"
+          # to access this envionment)
+          environment-name: protobuf
+          create-args: protobuf
+          init-shell: bash
```

Comment:
Is there any point using an environment?

Reply:
Conda/Mamba can only install packages into isolated environments, they cannot install into the system environment.

Reply:
Why do we need an environment on the temporary GH Actions runner?

Reply:
I am asking why are you using mamba/conda instead of pip (which is simpler - no `bash -el {0}` needed)?

Reply:
Mamba installs low-level dependencies properly, pip doesn't.